README.md

```diff
@@ -25,13 +25,18 @@ The table below lists the supported operating systems and Python versions requir
 | Red Hat* Enterprise Linux* 8, 64-bit                       | 3.6, 3.8                                             |
 | CentOS\* 7, 64-bit                                         | 3.6, 3.7, 3.8                                        |
 | macOS\* 10.15.x versions                                   | 3.6, 3.7, 3.8                                        |
-| Windows 10\*, 64-bit Pro, Enterprise or Education editions | 3.6, 3.7, 3.8                                        |
-| Windows Server\* 2016 or higher                            | 3.6, 3.7, 3.8                                        |
+| Windows 10\*, 64-bit Pro, Enterprise or Education editions | 3.6, 3.7, 3.8*                                        |
+| Windows Server\* 2016 or higher                            | 3.6, 3.7, 3.8*                                        |
+
+> \*_At the moment, For Python 3.8 on Windows, OpenVINO requires installation of [Microsoft Visual C++ Redistributable](https://visualstudio.microsoft.com/downloads/#microsoft-visual-c-redistributable-for-visual-studio-2019). This is not  required for Python 3.6 and 3.7, and will not be required for Python 3.8 in the next OpenVINO release._ 
 
 ## 📝 Installation Guide
 
 > **NOTE:** If OpenVINO is installed globally, please do not run any of these commands in a terminal where setupvars.bat or setupvars.sh are sourced. For Windows, we recommend using _Command Prompt (cmd.exe)_, not _PowerShell_.
 
+The [wiki](https://github.com/openvinotoolkit/openvino_notebooks/wiki#guides-per-operating-system) has more
+detailed information about required packages, and links to installers for Git and supported versions of Python.
```

Comment:
Since potentially thousands of visitors will come to this page during the Build conference (May 25-27), should we mention that Azure ML guide is in the wiki? (Even if this is just temporary)

Reply:
> Since potentially thousands of visitors will come to this page during the Build conference (May 25-27), should we mention that Azure ML guide is in the wiki? (Even if this is just temporary)

Yes, I'll add that.